rita/src/rita_exit/db_client/mod.rs

```diff
@@ -265,23 +338,22 @@ impl Handler<SetupClient> for DbClient {
 
                         let new_ip = incr_dummy(&conn)?;
 
-                        //TODO: Send email
-
                         let user_country = if SETTING.get_allowed_countries().is_empty() {
```

Comment:
should we mark this for removal in some version?

Reply:
I'm talking about the country code stuff. We don't send it in future versions right?

Reply:
Yeah good point, it should be removed fire alpha 6